content/docs/introduction/getting_started.md

```diff
@@ -25,15 +25,15 @@ Before starting Prometheus, let's configure it.
 
 ## Configuring Prometheus to monitor itself
 
-Prometheus collects metrics from monitored targets by scraping metrics HTTP
+Prometheus collects metrics from monitored targets by scraping metrics from HTTP
```

Comment:
This should be parsed as (scraping (metrics HTTP endpoints)), so it's fine as-is.

Reply:
Okay - will revert - was wondering if there should be a possessive in here somewhere then?

Reply:
The "on these targets" indicates the possession.